testdata/tail.txt

```diff
@@ -0,0 +1,79 @@
+mito -data state.json src.cel
+! stderr .
+cmp stdout want.txt
+
+-- state.json --
+{"list": [1, 2, 3, 4, 5]}
+-- src.cel --
+{
+	"lisp": {"head": state.list[?0], "tail": tail(state.list)},
+	"00": {"front": front(state.list, 0), "tail": tail(state.list, 0)},
+	"02": {"front": front(state.list, 2), "tail": tail(state.list, 2)},
```

Comment:
Can we add a test with `want_more: true` such that the list `list` gets exhausted?

Reply:
We can see that with the cases 05 and 10.

Reply:
Thanks!